.github/ISSUE_TEMPLATE/RUNNING-ISSUE-FORM.yml

```diff
@@ -0,0 +1,117 @@
+name: A problem while Sonic Pi is running 
+description: Sonic Pi starts up, but behaves unexpectedly during use.
+labels: ["triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hello! Thank you for wanting to raise an issue for us, so that we know what's not working for you.
+
+        Sorry you're having issues while Sonic Pi is running! We're always working to improve its behaviour.
+
+        The core Sonic Pi development team is fairly small and made up of volunteers (with the exception of Sam Aaron, who relies much on crowd-funding) so the more help we can receive to manage, troubleshoot and hopefully fix your issues for you the better - help us to help you! Please follow the checklist below - that way you might not even need to raise an issue after all :smile:
+  - type: checkboxes
+    id: read-troubleshooting-guide
+    attributes:
+      label: Prerequisites
+      options:
+        - label: I've read the [Sonic Pi troubleshooting guide](https://github.com/sonic-pi-net/sonic-pi/wiki/Troubleshooting-Issues) and followed its suggestions where relevant
+          required: true
+  - type: dropdown
+    id: sonic-pi-version
+    attributes:
+      label: Sonic Pi Version
+      description: |
+        What version of Sonic Pi were you running?
+        (Was it a release version, Beta, or a version you compiled from source yourself?)
+      options:
+        - Self compiled
```

Comment:
Will these lists need to be updated manually each time a new release is made? Might it be worth adding an "Other" option in case that ever gets forgotten?